docs/TESTING.md

```diff
@@ -17,28 +17,64 @@ docker run -it \
            --volume $(pwd):/linux-loader \
            rustvmm/dev:v${container_version}
 cd linux-loader/
-cargo test
+cargo test --all-features
 ```
 
-### bzImage test
+## Test Images
 
-As we don't want to distribute an entire kernel bzImage, the `load_bzImage`
-test is ignored by default. In order to test the bzImage support, one needs to
-locally build a bzImage, copy it to the `src/loader` directory and run
-`cargo test`:
+The kernel images used in the unit tests fall into 3 categories:
+- downloaded from known distro sources using the
+  [download script](../.buildkite/download_resources.sh).
+- fuzzing images; these images were generated by fuzzers such as `libfuzzer` or
+  `afl` starting from an existing image already used by the unit tests. These
+  are named with `fuzz_` so that they are easy to identify.
+- generated images (either by building the Linux kernel, or by using other
+  tools such as [ELFIO](https://github.com/serge1/ELFIO)).
+
+### Building the `PE` image
+
+The PE image used by the linux-loader unit tests is built as follows:
 
 ```bash
-# Assuming your linux-loader and linux-stable are both under ${LINUX_LOADER}:
-git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git ${LINUX_LOADER}/linux-stable
-cd linux-stable
-make bzImage
-cp linux-stable/arch/x86/boot/bzImage ${LINUX_LOADER}/linux-loader/src/loader/
-cd ${LINUX_LOADER}/linux-loader
-container_version=11
-docker run -it \
-           --security-opt seccomp=unconfined \
-           --volume $(pwd):/linux-loader \
-           rustvmm/dev:v${container_version}
-cd linux-loader/
-cargo test
+git clone git://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+git checkout v4.20
+
+# To be able to run the following command, you'll need to install
+# flex and bison.
+make allnoconfig
+make Image
```

Comment:
I don't know if it's because of my setup, but this only worked for me on an `aarch64` machine. Can you `make Image` on x86_64?

Reply:
No, this is only for aarch64. The PE image is only used on aarch64 as well.

Reply:
I explicitly called out that these steps work on aarch64.

Reply:
The `make Image` rule works on `x86_64` but requires cross compilation. Should we mention that these steps work out of the box only on aarch64 machines?

Reply:
Looks like Andreea added an explicit mention: `The PE image used by the linux-loader unit tests is built on an aarch64
machine using the following commands:...`